docker/bitmask-docker.sh

```diff
@@ -81,7 +81,9 @@ help() {
     echo ">> Bitmask on docker"
     echo "Run the bitmask app in a docker container."
     echo
-    echo "Usage: $0 {init bitmask.json | update | run | help }"
+    echo "Usage: $0 {init | update} [bitmask-nightly.json] {run | help}"
+    echo        
+    echo "  ?.json : The bitmask*.json file describes the version that will be used for each repo."
```

Comment:
Adding this here looks good. You should remove the explanation right after `init` doc.